src/functions-reference/bounded_discrete_distributions.Rmd

```diff
@@ -339,6 +418,73 @@ eta, and cutpoints c.
+
+`y ~ ` **`ordered_logistic_glm`**`(x, beta, c)`
+
+Increment target log probability density with `ordered_logistic_lpmf(y | x, beta, c)`
+dropping constant additive terms.
+<!-- real; ordered_logistic ~; -->
+\index{{\tt \bfseries ordered\_logistic\_glm }!sampling statement|hyperpage}
+
+### Stan Functions
+
+<!-- real; ordered_logistic_glm_lpmf; (int y | row_vector x, vector beta, vector c); -->
+\index{{\tt \bfseries ordered\_logistic\_glm\_lpmf }!{\tt (int y \textbar\ row\_vector x, vector beta, vector c): real}|hyperpage}
+
+`real` **`ordered_logistic_glm_lpmf`**`(int y | row_vector x, vector beta, vector c)`<br>\newline
+The log ordered logistic probability mass of y, given linear predictors `x * beta`, and cutpoints c. 
+The size of independent variable row vector `x` needs to match the size of the weight vector `beta`. 
+Cutpoints `c` must be ordered.
+
+<!-- real; ordered_logistic_glm_lpmf; (int y | matrix x, vector beta, vector c); -->
+\index{{\tt \bfseries ordered\_logistic\_glm\_lpmf }!{\tt (int y \textbar\ matrix x, vector beta, vector c): real}|hyperpage}
+
+`real` **`ordered_logistic_glm_lpmf`**`(int y | matrix x, vector beta, vector c)`<br>\newline
+The log ordered logistic probability mass of y, given linear predictors `x * beta`, and cutpoints c. 
+Same value of independent variable `y` is used for all instances.
+The number of columns of independent variable row vector `x` needs to match the size of the weight vector `beta`.
+Cutpoints `c` must be ordered. If `x` and `y` are data (not parameters) this function can be executed on a GPU.
+
+<!-- real; ordered_logistic_glm_lpmf; (int[] y | row_vector x, vector beta, vector c); -->
+\index{{\tt \bfseries ordered\_logistic\_glm\_lpmf }!{\tt (int[] y \textbar\ row\_vector x, vector beta, vector c): real}|hyperpage}
+
+`real` **`ordered_logistic_glm_lpmf`**`(int[] y | row_vector x, vector beta, vector c)`<br>\newline
+The log ordered logistic probability mass of y, given linear predictors `x * beta`, and cutpoints c. 
+Same row vector of independent variables `x` is used for all instances.
+The size of independent variable row vector `x` needs to match the size of the weight vector `beta`. 
+Cutpoints `c` must be ordered.
+
+<!-- real; ordered_logistic_glm_lpmf; (int[] y | matrix x, vector beta, vector c); -->
+\index{{\tt \bfseries ordered\_logistic\_glm\_lpmf }!{\tt (int[] y \textbar\ matrix x, vector beta, vector c): real}|hyperpage}
+
+`real` **`ordered_logistic_glm_lpmf`**`(int[] y | matrix x, vector beta, vector c)`<br>\newline
+The log ordered logistic probability mass of y, given linear predictors
+`x * beta`, and cutpoints c. 
+The number of rows of the independent variable matrix `x` needs to match the length of the dependent variable vector `y`.
+The number of columns of independent variable row vector `x` needs to match the size of the weight vector `beta`.
+Cutpoints `c` must be ordered. If `x` and `y` are data (not parameters) this function can be executed on a GPU.
```

Comment:
of independent -> of the independent
Cutpoints `c` -> The cutpoint vector `c`

Reply:
fixed